pkg/infinity/postprocess.go

```diff
@@ -39,11 +39,6 @@ func PostProcessFrame(ctx context.Context, frame *data.Frame, query models.Query
 		return transformations.GetSummaryFrame(frame, query.SummarizeExpression, query.SummarizeBy, alias)
 	}
 	frame.Meta = &data.FrameMeta{Custom: &CustomMeta{Query: query}}
-	if err != nil {
```

Comment:
was this deleted accidentally? it doesn't look like its related to the migration